### DISCUSSION TITLE
Understanding problem when converting to physical model

### DISCUSSION THREAD

**Discussion by Quietscheente** [OP-TED/eForms-SDK#660] [Q&A] [answered]
Hello,

I am having trouble understanding part of the conversion principle.
When I try to validate the physical model I get an error that a mandatory attribute "schemeName" is missing. in a ID-node.

In the fields.json there is a `"xpathRelative" : "cbc:ID[@schemeName='notice-id']",`. 
As I understood it, it says: element ID, but only the one with the attribute value schemeName='notice-id'. So I searched further with the expectation to find somewhere in fields.json an xpath ending with @schemeName, so the place where the value is set. But I can't find an element for that. Where is my thinking error? Or rather, at which point should the schemeName attribute be set?

**Comment by YvesJo** [comment, marked as answer]
Hi,
Not all attributes have a field associated. Need for attributes could be detected by looking at the Field and The XML element type. Few fields were defined when they corresponded to some defined Business Terms.
Starting SDK 1.9, you will be able to find in fields.json a field for every single attribute.
